### PR TITLE
fix two typo

### PR DESCRIPTION
**Pull request by zhustec**

### CODE REVIEW

File: lib/kimurai/base_helper.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def escape_url(url)
     rescue URI::InvalidURIError => e
       URI.parse(URI.escape url).to_s rescue url
     else
-      url
```
Comment (vifreefly) [inline lib/kimurai/base_helper.rb]: This is actually not a bug, the method should return string object (`url`) not URI `uri` if url is correct

File: lib/kimurai/base/storage.rb [hunk 2]
```diff
@@ -84,15 +84,15 @@ def clear!
               database.roots.each { |key| database.delete key }
             end
           else
-            database = {}
```
Comment (vifreefly) [inline lib/kimurai/base/storage.rb]: Good catch, thanks!

File: lib/kimurai/base/storage.rb [hunk 3]
```diff
@@ -84,15 +84,15 @@ def clear!
...
+            @database = {}
           end
         end
       end
 
       def delete!
         @mutex.synchronize do
           if path
-            File.delete path if File.exists? path
+            File.delete path if File.exist? path
```
Comment (vifreefly) [inline lib/kimurai/base/storage.rb]: You're right, thanks!

**Comment by zhustec** [comment]
Thanks for your detail explanation, I will push a new commit to fix it.

**Comment by vifreefly** [comment]
@zhustec  Thanks, merged!

**Pull request merged**